ssd1306/ssd1306.c

```diff
@@ -16,7 +16,11 @@ void ssd1306_WriteCommand(uint8_t byte) {
 
 // Send data
 void ssd1306_WriteData(uint8_t* buffer, size_t buff_size) {
+#if defined(SSD1306_USE_DMA)
+    HAL_I2C_Mem_Write_DMA(&SSD1306_I2C_PORT, SSD1306_I2C_ADDR, 0x40, 1, buffer, buff_size);
+#else
     HAL_I2C_Mem_Write(&SSD1306_I2C_PORT, SSD1306_I2C_ADDR, 0x40, 1, buffer, buff_size, HAL_MAX_DELAY);
+#endif
```

Comment:
Shouldn't corresponding _WriteCommand be changed as well?

Reply:
`ssd1306_WriteCommand()` is used to transfer only one byte each time. If use DMA in `ssd1306_WriteCommand()`, CPU need to read the byte's address and transfer the address to DMA, then DMA use the address to transmit only one byte to I2C.
CPU still need to read every byte, DMA's effect isn't good here.
CPU transmit to DMA, then DMA transmit to I2C, maybe slower than CPU directly transmit to I2C.
But when transmit frame buffer, CPU only need to tell DMA start address and size of buffer, only 2 variables, then DMA can transmit these 128*64 bytes without CPU's help. DMA's effect is good here.

If want DMA to show its advantage, need to place all command bytes needed by an operate ( such as tens of the command bytes in `ssd1306_Init()` ) in an array, then use `HAL_I2C_Mem_Write_DMA()` only one time to transfer the whole array.

So, I prefer not using DMA in `ssd1306_WriteCommand()`.